ocs_ci/ocs/cluster.py

```diff
@@ -2635,6 +2636,76 @@ def exec_cmd_on_cluster_node(self, cmd):
         return_stdout = return_output[1]
         return return_stdout
 
+    def get_thin_provisioning_alerts(self):
+        """
+        Get the list of alerts that active in the cluster
+
+        Returns:
+            list: alrets name
+
+        """
+        new_prom = PrometheusAPI()
+
+        alert_full = new_prom.get("alerts")
+        alerts_data = alert_full.json().get("data").get("alerts")
+        alerts_names = list()
+        for entity in alerts_data:
+            logger.debug(entity.get("labels").get("alertname"))
+            alerts_names.append(entity.get("labels").get("alertname"))
+
+        return alerts_names
+
+    def check_for_alert(self, alert_name):
+        """
+        Check to see if a given alert is available
+
+        Args:
+            alert_name (str): Alert name
+
+        Returns:
+            bool: True if alert is available else False
+
+        """
+        if alert_name in self.get_thin_provisioning_alerts():
+
+            return True
+
+        return False
+
+    def parse_topolvm_metrics(self, metrics):
+        """
+        Returns the name and value of topolvm metrics
+
+        Args:
+            metric_name (list): metrics name to be paesed
+
+        Returns:
+            dict: topolvm metrics by: names: value
+        """
+        new_prom = PrometheusAPI()
+        metrics_short = dict()
+
+        for metric_name in metrics:
+            metric_full = new_prom.query(metric_name)
+            metric_value = metric_full[0].get("value")[1]
+            logger.info(f"{metric_name} : {metric_value}")
+            metrics_short[metric_name] = metric_value
+
+        return metrics_short
+
+    def validate_thin_pool_data_percent_metric(self):
+        os_data_precent = self.get_thin_pool1_data_percent()
```

Comment:
Please change the name of the variable to thin_pool_data_percent or something that will more clear.